runtime/common/src/claims.rs

```diff
@@ -1444,9 +1445,12 @@ mod benchmarking {
 			assert_eq!(Claims::<T>::get(eth_address), Some(VALUE.into()));
 			let source = sp_runtime::transaction_validity::TransactionSource::External;
 			let call = Call::<T>::claim { dest: account.clone(), ethereum_signature: signature.clone() };
+			let call_enc = call.encode();
 		}: {
 			super::Pallet::<T>::validate_unsigned(source, &call).map_err(|e| -> &'static str { e.into() })?;
-			super::Pallet::<T>::claim(RawOrigin::None.into(), account, signature)?;
+			<Call<T> as Decode>::decode(&mut &*call_enc)
+				.expect("call is encoded above, encoding must be correct")
+				.dispatch_bypass_filter(RawOrigin::None.into())?;
```

Comment:
maybe the benchmark would look more logical if we do:
* decode call
* check validate_unsigned for the call
* dispatch the call.

but both looks fine.